scheduler/test/cook/test/kubernetes/api.clj

```diff
@@ -79,19 +79,19 @@
         (let [workdir-volume (->> pod
                                   .getSpec
                                   .getVolumes
-                                  (filter (fn [^V1Volume v] (= "cook-workdir" (.getName v))))
+                                  (filter (fn [^V1Volume v] (= "cook-workdir-volume" (.getName v))))
                                   first)]
           (is (not (nil? (.getEmptyDir workdir-volume)))))
 
         (let [^V1Container container (-> pod .getSpec .getContainers first)]
           (is (= "required-cook-job-container" (.getName container)))
           (is (= ["/bin/sh" "-c" "foo && bar"] (.getCommand container)))
           (is (= "alpine:latest" (.getImage container)))
-          (is (= 3 (count (.getEnv container))))
+          (is (= 4 (count (.getEnv container))))
           (is (= "/mnt/sandbox" (.getWorkingDir container)))
           (let [workdir-mount (->> container
```

Comment:
Why workdir-mount here and workdir-volume above?

Reply:
those are different